src/protocols/bdx/BDXMessageUtils.cpp

```diff
@@ -185,22 +158,35 @@ CHIP_ERROR TransferInit::Parse(System::PacketBuffer * aBuffer, TransferInit & aP
 
     SuccessOrExit(bufReader.Read16(&aParsedMessage.mFileDesLength).StatusCode());
 
-    VerifyOrExit(bufReader.OctetsRead() + aParsedMessage.mFileDesLength <= aBuffer->DataLength(),
+    VerifyOrExit(bufReader.OctetsRead() + aParsedMessage.mFileDesLength <= aBuffer.DataLength(),
```

Comment:
Please parenthesize subexpression before the `<=` here and elsewhere in this function where `bufReader.OctetsRead() + XXX` is used.

Reply:
ack